app/assets/stylesheets/global/_layout.scss

```diff
@@ -2,6 +2,10 @@ header {
   background-color: $header-background-color;
 }
 
+body {
+  text-align: #{$left};
```

Comment:
Can this just be `text-align: $left` as below? I think the `#{...}` syntax is just for keys?